[BACK]/app/controllers/User.js

```diff
@@ -46,7 +88,7 @@ exports.register = async function (req, res) {
                             createdAt: new Date(),
                             updatedAt: new Date(),
                             RoleId: 1,
-                            TeamId: newUser.team
+                            TeamId: newUser.teamId
```

Comment:
Change front request

Reply:
I updated back end code `teamId` => `team` to not make front not working, but it seems `team` not use `id` 🤔